rust/src/delta_datafusion.rs

```diff
@@ -797,14 +771,11 @@ impl PhysicalExtensionCodec for DeltaPhysicalCodec {
     fn try_encode(
         &self,
         node: Arc<dyn ExecutionPlan>,
-        buf: &mut Vec<u8>,
+        _buf: &mut Vec<u8>,
```

Comment:
Yeah, that could be true. I'm not sure whether something other than `DeltaScan` can be passed here, in which case this wouldn't be a no-op due to the error returned.

Arguably there's no need to even have `DeltaScan` now, since it's just an empty wrapper around `ParquetExec`, but I left it thinking something will be needed here sooner or later (e.g. column mapping mentioned in the TODO)